src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.split(';');
+  const splitedList = stylesList.map(
+    style => style.split(':').map(prop => prop.trim()
+    )
+  );
+
+  const filteredList = splitedList.filter(item => item.length >= 2);
+  const allStylesMap = filteredList.reduce((cssStyles, [key, value]) => ({
+    ...cssStyles,
+    [key]: value,
+  }), {});
+
+  return allStylesMap;
```

Comment:
```suggestion
  return filteredList.reduce((cssStyles, [key, value]) => ({
    ...cssStyles,
    [key]: value,
  }), {});
```

Reply:
Thanks. I realized. I just understand better when I first do everything in a variable and then return it. I will be corrected)